pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1684,29 +1684,6 @@ func doActivateTail(ctx *domainContext, status *types.DomainStatus,
 		status.UUIDandVersion, status.DisplayName)
 }
 
-// VLAN filtering could have been enabled on the bridge immediately after
```

Comment:
Is this no longer needed due to some unknown netlink code? (New kernel version??)

Reply:
Some time ago, this was moved from domainmgr to zedrouter (NI Reconciler): https://github.com/lf-edge/eve/blob/master/pkg/pillar/nireconciler/linuxitems/vlanbridge.go#L106-L131
I just forgot to remove this no longer used function back then.